src/esp/bindings_js/modules/simenv_embind.js

```diff
@@ -19,10 +19,14 @@ class SimEnv {
    * @param {Object} episode - episode to run
    * @param {number} agentId - default agent id
    */
-  constructor(config, episode, agentId) {
+  constructor(config, episode = {}, agentId = 0) {
```

Comment:
Why not just use null as the default value? That way you would only have to check for falsiness.

Reply:
I was thinking of future changes. Episode also contains goal, so it might not be sufficient to set it to null yet. So, setting it to object allows us to assume there will always be an object and then we can do object level compares.

Reply:
For e.g. I won't be able to do `episode.goal` without check if episode is null. (That would be one extra check if default value is not `{}`)